02_configure_host.sh

```diff
@@ -104,6 +107,12 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -i vm-setup/inventory.ini \
     -b -vvv vm-setup/firewall.yml
 
+# FIXME(stbenjam): ansbile firewalld module doesn't seem to be doing the right thing
+if [ "$USE_FIREWALLD" == "True" ]; then
+  sudo firewall-cmd --zone=libvirt --change-interface=provisioning
+  sudo firewall-cmd --zone=libvirt --change-interface=baremetal
```

Comment:
I needed this on RHEL8, needs more investigation, but after the ansible run, I see in the logs it added baremetal and provisioning to the libvirt zone, but only the last one is there.